ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -5326,6 +5328,15 @@ PMM:
   implements:
   - MONITORING
 
+PMM2X:
+  description: "Particulate matter monitoring for 2 zones."
```

Comment:
The fields you have defined describe two separate sensors of different types, not the same type of sensor duplicated across multiple zones. For example, this is what i would expect if it was two zones:
```
uses:
- zone_air_pm2pt5_concentration_sensor_1
- zone_air_pm2pt5_concentration_sensor_2
```